src/main/drivers/nvic.h

```diff
@@ -28,11 +28,14 @@
 #define NVIC_PRIO_SONAR_EXTI               NVIC_BUILD_PRIORITY(2, 0)  // maybe increase slightly
 #define NVIC_PRIO_DSHOT_DMA                NVIC_BUILD_PRIORITY(2, 1)
 #define NVIC_PRIO_TRANSPONDER_DMA          NVIC_BUILD_PRIORITY(3, 0)
-#define NVIC_PRIO_RX_INT_EXTI              NVIC_BUILD_PRIORITY(0x0f, 0x0f)
-#define NVIC_PRIO_RX_BUSY_EXTI             NVIC_BUILD_PRIORITY(3, 0)
+
+// RX_SPI must be lower priority than SPI DMA so EXTI ISRs don't interfere with SPI transfers and transfer complete callbacks
+#define NVIC_PRIO_RX_SPI_INT_EXTI          NVIC_BUILD_PRIORITY(3, 0x0f)
+#define NVIC_PRIO_RX_INT_EXTI              NVIC_BUILD_PRIORITY(3, 0x0f)
+#define NVIC_PRIO_RX_BUSY_EXTI             NVIC_BUILD_PRIORITY(3, 0x0f)
+
```

Comment:
Wasn't it already the case that the DMA is higher priority with the SPI DMA having a base priority of 0, which is lower than 3?
```
#define NVIC_PRIO_SPI_DMA                  NVIC_BUILD_PRIORITY(0, 0)
```

Reply:
one of them was 0, one of them was 3.

I think there's still an error in the nvic config as some have a PRIORITY of 0x0f which needs 4 bits, which and the system is configured to use 2.  a GROUP of 0x0f is ok, but with only 2 bits the max PRIORITY is 3.

I think part of the issue was that I was seeing 0x00 written to the NVIC registers so I fixed the ones that were a problem for the specific case I was fixing.

Feel free to fix the other priorities accordingly in a new PR.